packages/@o3r/rules-engine/schematics/ng-update/index.ts

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+/* eslint-disable camelcase */
```

Comment:
why do you need these eslint exceptions for the whole file?

Reply:
Do not need any, they came probably from the copied original file.
Removed it